### ISSUE TITLE
421: Misdirected RequestTransmission when connecting from other host than locahost

### ISSUE THREAD

**Issue by pagaille** [stephdl/nethserver-transmission#4]
Hi Steph,

I reveive 421: Misdirected RequestTransmission received your request, but the hostname was unrecognized.To fix this, choose one of the following options:Enable password authentication, then any hostname is allowed.Add the hostname you want to use to the whitelist in settings.If you're editing settings.json, see the 'rpc-host-whitelist' and 'rpc-host-whitelist-enabled' entries.This requirement has been added to help prevent DNS Rebinding attacks.

when first opening the web app. 

I had to stop the service then add my hostname to the entry "rpc-host-whitelist" in /var/lib/transmission/.config/transmission-daemon/settings.json

**Comment by stephdl** [comment]
I cannot reproduce :-?

what version of transmission do you use ?

**Comment by stephdl** [comment]
[some clues ](https://github.com/transmission/transmission/wiki/Editing-Configuration-Files#rpc)

**Comment by stephdl** [comment]
could you test : 

`yum install http://mirror.de-labrusse.fr/NethDev/nethserver-transmission/nethserver-transmission-1.1.9-1.ns7.noarch.rpm`

then `*.DomainName,DomainName` is set by default
add your custom domain name by 

`config setprop transmission rpcWatchedList 'toto.com,tata.com,tutu.fr'`
`signal-event nethserver-transmission-save`

check if no errors in /var/log/messages
check if the good settings are in settings.json

**Comment by pagaille** [comment]
Thanks Steph,

I cannot test because I customised my installation to use a nethserver-managed username to give it access to an ibay (and be able to handle the files through Samba) and therefore the user home path  changed from /var/lib/transmission/ to /var/lib/nethserver/home/transmission/. --> I had to copy the settings.json to that new place and I guess that it breaks your module.

However if you modify the rpc-host-whitelist entry with the hostname of the server it should work. That's what I did.
